src/layouts/VerticalLayout.tsx

```diff
@@ -13,31 +13,20 @@ export function VerticalLayout({
   renderers,
   cells,
 }: LayoutProps) {
-  const verticalLayout = uischema as VerticalLayoutUISchema<unknown>
-  const groupLayout = uischema as GroupLayout
+  const { elements } = uischema as VerticalLayoutUISchema<unknown>
```

Comment:
Does this still require a cast, or could the cast be removed?

Reply:
it does, without the cast the type is `UISchemaElement` and doesn't have `elements` as a property.

Reply:
> Does this still require a cast, or could the cast be removed?

it does, without the cast the type is UISchemaElement and doesn't have elements as a property.